.lando.yml

```diff
@@ -13,7 +13,7 @@ config:
 services:
   appserver:
     type: php:7.2
-    xdebug: "debug,develop"
+    # xdebug: "debug,develop"
```

Comment:
I just disabled this since I'm not using PHPStorm or X-debug

Reply:
Got it, makes sense.